src/tools/bin/tbg

```diff
@@ -250,6 +250,7 @@ help()
     echo "                                 spaces within the right side of assign are not allowed"
     echo "                                 e.g. -o \"VARNAME1=10 VARNAME2=5\""
     echo "                                 Overwriting is done after cfg file was executed"
+    echo "-f | --force                   - Start new job even if TBG_dstPath exists."
```

Comment:
I would replace `Start new job` with `Overwrite` since this is independent of the `-s` option which does submission/job starts

Reply:
in line 239 the `[-f` needs to be added before the `[-h]`